bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +417,175 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+from . import pythonization
+from .._numbadeclare import _NumbaDeclareDecorator
+import re
+import warnings
+import numpy as np
+
+# Pythonizer 
+# def pythonize_define():
+    # @pythonization('RDataFrame', ns = 'ROOT')
+    # def pythonizer_rdf(klass):
+    #     klass._OriginalDefine =  klass.Define
+    #     klass.Define = _PyDefine
+
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
```

Comment:
Why did you add separate pythonizor functions for define and filter? We usually place all the pythonizations in one pythonizor (from which helper functions are invoked). This way is more efficient, since each new pythonizor requires one more check once a new class is loaded (to see if the new class name matches RDF in this case) - if we just use one pythonizor for all RDF pythonizations, just one check will be done.